AlpsSDK/ModelExtensions.swift

```diff
@@ -103,3 +104,14 @@ extension ProximityEvent {
         self.distance = distance
     }
 }
+
+extension CLProximity {
+    
+    static let allValues = [unknown, immediate, near, far]
+}
+
+extension Date {
+    func nowTimeInterval() -> Int64 {
```

Comment:
We have to be careful with types here.

In general ios uses NSTimeInterval which is a double.
On backend they use Long Int.

For now it seems fair to use Int64. But if we ever experience any bugs related to it - we should see what's most consistent.

Reply:
Ok, I did choose Int64 because of 
```guard let proximityEventCreatedAt = proximityEvent.createdAt else {return}```
proximityEventCreatedAt is implicitly an Int64